src/SettingStore.php

```diff
@@ -48,6 +48,29 @@ public function get($key, $default = null)
 	}
 
 	/**
+	 * Get a flattened array from the settings data.
+	 *
+	 * @param  array   $array   Optional array of settings values, with a fallback to all settings.
+	 * @param  string  $prefix  Optional prefix.
+	 *
+	 * @return array
+	 */
+	public function getRawData($array = null, $prefix = null)
```

Comment:
I think the function should be named "getFlattenedData" instead, as "raw" doesn't really convey what it does.

Reply:
Actually, just `flattened` would probably be best, to be consistent with other method names (`all` and `get`).